include/superior_mysqlpp/logging.hpp

```diff
@@ -63,6 +38,27 @@ namespace SuperiorMySqlpp {
          * so you shall either implement all logging functions or only those you want.
          */
 
+        namespace detail {
+
+            /**
+             * Global variable guaranteeing single access to stderr from standard implementation of loggers
+             * @remark Conceptally should be hidden in #logStderr function, but that would lead to multiple instances.
+             */
+            std::atomic_flag stderrLogWriteLock{};
```

Comment:
Global variables are usually warning sing, so I'm pointing it out. My previous solution passed `atomic_lock`s around as they used previously, this one leads to less boilerplate with no obvious disadvantage for me.

Reply:
My bad, this rewrite looks nicer then original approach, but global variables in header-only lib are no-no, of course.